src/components/rich-text-editor/rich-text-editor-options.ts

```diff
@@ -9,7 +9,7 @@ export abstract class MRichTextEditorDefaultOptions {
     public scrollableContainer: string | undefined;
     public toolbarStickyOffset: number = 0;
     public pluginsEnabled: string[] = ['align', 'draggable', 'embedly', 'entities', 'file', 'fontFamily', 'fontSize', 'fullscreen', 'inlineStyle', 'lineBreaker', 'link', 'lists', 'paragraphFormat', 'paragraphStyle', 'quote', 'save', 'specialCharacters', 'table', 'url', 'wordPaste', 'stylesSubMenu', 'listesSubMenu', 'insertionsSubMenu'];
-    public wordPasteModal: boolean = false;
+    public wordPasteWindow: boolean = false;
```

Comment:
NOOOOPPEEEE.  Faut que ça respecte l'API de Froala pis le nom de lap ropriété c'est wordPasteModal :)

Reply:
Good catch 👍

Faudrait bien vérifier qu'il n'y a pas d'autre erreur de find and replace 😅

Reply:
fixed